poc_iot_verifier/src/error.rs

```diff
@@ -48,6 +48,12 @@ pub enum Error {
     InvalidExponent(String),
```

Comment:
I’m hoping this gets in first but yes, I noted in a call the other day I intentionally did this PR in the older error handling style because it was going to be a big enough change without doing any switching to anyhow; at least if that PR gets merged first the rebase will still keep the changes isolated from this merge

Reply:
it didn't :(